mem0/llms/gemini.py

```diff
@@ -44,11 +45,9 @@ def _parse_response(self, response, tools):
 
             for part in response.candidates[0].content.parts:
                 if fn := part.function_call:
+                    fn_call = type(fn).to_dict(fn)
```

Comment:
The old syntax doesn't work properly with some cases of functions with deep values ​​or iterations. The update is referenced this way: https://ai.google.dev/gemini-api/tutorials/extract_structured_data